src/modules/launcher/Wox.Plugin/Result.cs

```diff
@@ -76,7 +78,17 @@ public ToolTipData ToolTipData
         /// <summary>
         /// Gets or sets the text that will get displayed in the Search text box, when this item is selected in the result list.
         /// </summary>
-        public string QueryTextDisplay { get; set; }
+        public string QueryTextDisplay
+        {
+            get => _queryTextDisplay;
+            set
+            {
+                if (Microsoft.PowerToys.Settings.UI.Library.PowerLauncherProperties.SearchResultChangesQueryText || OriginQuery.ActionKeyword != string.Empty)
```

Comment:
Unsure how to read PowerLauncherProperties, where to find the instance to read from?